phpcs.xml

```diff
@@ -1,15 +1,77 @@
 <?xml version="1.0"?>
-<ruleset name="WordPress Coding Standards for Themes">
-	<description>Generally-applicable sniffs for WordPress themes</description>
+<ruleset name="go">
+	<description>Rules for go</description>
 
-	<rule ref="WordPress">
-		<exclude name="Squiz.PHP.CommentedOutCode.Found"></exclude>
-		<exclude name="Squiz.PHP.NonExecutableCode.Unreachable"></exclude>
+    <arg name="extensions" value="php" />
+    <arg name="basepath" value="./" />
+    <file>./</file>
+
+    <exclude-pattern>*/build/*</exclude-pattern>
+    <exclude-pattern>*/dist/*</exclude-pattern>
+    <exclude-pattern>*/vendor/*</exclude-pattern>
+    <exclude-pattern>*/node_modules/*</exclude-pattern>
+	<exclude-pattern>*/wordpress*/*</exclude-pattern>
+    <exclude-pattern>*/\.*</exclude-pattern>
+    <exclude-pattern>*/.dev/*</exclude-pattern>
+
+	<arg name="colors"/>
+	<arg value="sp"/>
+
+	<!-- Include the WordPress-Extra standard. -->
+	<rule ref="WordPress-Extra">
+		<!--
+		We may want a middle ground though. The best way to do this is add the
+		entire ruleset, then rule by rule, remove ones that don't suit a project.
+		We can do this by running `phpcs` with the '-s' flag, which allows us to
+		see the names of the sniffs reporting errors.
+		Once we know the sniff names, we can opt to exclude sniffs which don't
+		suit our project like so.
+		-->
+		<exclude name="WordPress.Files.FileName.InvalidClassFileName"/>
+		<exclude name="WordPress.Files.FileName.NotHyphenatedLowercase"/>
+		<!--
+		ShortPrefixPassed is excluded due to the use of prefix `Go` This prefix, 
+		while short in length, is approved by the themes team at WordPress.org.
+		For this reason the rule should be excluded as we are not concerned with 
+		code collisions.
+		-->
+		<exclude name="WordPress.NamingConventions.PrefixAllGlobals.ShortPrefixPassed" />
```

Comment:
Added this snipped to ignore the short prefix error thrown by PHPCS. The other route we can go with this is to increase the length of the prefix. Maybe something like `gdgo`?

Reply:
I ignored those in all of the Primer themes as well. It doesn't seem worth it to go through and append a prefix onto every method. We're also namespacing our files so there shouldn't be conflicts with plugins.